gosdk/appchain.go

```diff
@@ -25,105 +23,77 @@ import (
 	"github.com/0xAtelerix/sdk/gosdk/utility"
 )
 
-func WithRootCalculator[STI StateTransitionInterface[AppTx, R],
-	AppTx apptypes.AppTransaction[R],
-	R apptypes.Receipt,
-	AppBlock apptypes.AppchainBlock](rc apptypes.RootCalculator) func(a *Appchain[STI, AppTx, R, AppBlock]) {
-	return func(a *Appchain[STI, AppTx, R, AppBlock]) {
-		a.rootCalculator = rc
-	}
-}
+// Default configuration for appchain services.
+const (
+	DefaultDataDir     = "/data"
```

Comment:
It should be part of types as tags or part of config manager

Reply:
Will have config manager as separate task